x-pack/plugins/security/public/session/session_timeout.test.ts

```diff
@@ -164,6 +164,19 @@ describe('SessionTimeout', () => {
     expect(http.fetch).toHaveBeenCalledTimes(1);
   });
 
+  it('exponentially increases retry time when extending session fails', async () => {
+    const { sessionTimeout, http } = createSessionTimeout(60 * 60 * 1000);
+    http.fetch.mockRejectedValue(new Error('Failure'));
+    await sessionTimeout.start();
+    expect(http.fetch).toHaveBeenCalledTimes(1);
+
+    // Increment system time enough so that session extension gets triggered
+    nowMock.mockReturnValue(Date.now() + SESSION_EXTENSION_THROTTLE_MS + 10);
+    window.dispatchEvent(new Event('mousemove'));
+
+    expect(http.fetch).toHaveBeenCalledTimes(1);
+  });
```

Comment:
When I remove your changes in `session_timeout.ts`, this test doesn't seem to fail. I played around with it for a bit but I'm not sure what's going wrong. Can you take another look to make sure the test is correct?